src/librustc_mir/build/matches/simplify.rs

```diff
@@ -115,14 +116,14 @@ impl<'a, 'gcx, 'tcx> Builder<'a, 'gcx, 'tcx> {
                     ty::Int(ity) => {
                         // FIXME(49937): refactor these bit manipulations into interpret.
                         let size = Integer::from_attr(&tcx, SignedInt(ity)).size();
-                        let max = !0u128 >> (128 - size.bits());
+                        let max = mask(size);
```

Comment:
this could be `truncate(u128::max_value(), size)`

Reply:
Thank you for your review. Then, should I replace `mask` with `truncate` and remove the definition of `mask`?

Reply:
yes, I think that's for the best, unless you see a lot of other sites in the compiler where using `mask` would make sense.